### ISSUE TITLE
Aktuellen Strompreis anzeigen

### ISSUE THREAD

**Issue by bennysweb** [solectrus/solectrus#3418]
### Beschreibung (auf Deutsch oder Englisch)

Man könnte (ähnlich wie beim Auto-SoC) den aktuellen Strompreis im Dashboard anzeigen. Bei einem dynamischen Tarif (Tibber o.ä.) ändert sich der Wert stündlich und wird sowieso vom Tibber-Collector eingefangen (oder kann vom MQTT-Collector bereitgestellt werden), beim statischen Stromtarif ändert sich der Wert nicht und wird manuell vom User in den Einstellungen hinterlegt.

**Comment by ledermann** [comment]
Ja, das ist sinnvoll - aber ich meine, dass erst #2150 gemacht werden sollte. Andernfalls würde das zu Missverständnissen führen, da SOLECTRUS derzeitig die Kosten ausschließlich mit den statischen Preisen (die in den Einstellungen erfasst werden) errechnet.
